template/_package.json

```diff
@@ -43,8 +43,7 @@
     "@adonisjs/shield": "^1.0.4"<% } %><% if (ui === 'bootstrap') { %>,
     "bootstrap-vue": "^2.0.0-rc.11",
     "bootstrap": "^4.1.3"<% } else if (ui === 'vuetify') { %>,
-    "vuetify": "^1.5.5",
-    "vuetify-loader": "^1.2.1"<% } else if (ui === 'bulma') { %>,
+    "@nuxtjs/vuetify": "latest"<% } else if (ui === 'bulma') { %>,
```

Comment:
@pi0 do we want to use latest? I think it should be consistent with all other packages, so either latest for all or specific version for all

Reply:
Agree, @takehiro-takahashi could you please change it to the latest stable version  ?